### PR TITLE
Fixes, new views and helpers for Container entity

### PR DESCRIPTION
**Pull request by abalakh**
Test results in https://github.com/SatelliteQE/robottelo/pull/6773#issue-256759373

### CODE REVIEW

File: airgun/entities/container.py [hunk 1]
```diff
@@ -19,11 +19,44 @@ def search(self, value):
...
+        view = self.navigate_to(self, 'Details', entity_name=entity_name)
+        if power in (True, 'On'):
+            view.power_on.click()
+        elif power in (False, 'Off'):
+            view.power_off.click(handle_alert=True)
+        else:
+            raise ValueError(
+                'Wrong `power` value passed. Supported values are True/False/"On"/"Off"')
+        view.flash.assert_no_error(
+            ignore_messages=['Error - wrong number of arguments (given 1, expected 0)'])
```
Comment (oshtaier) [inline airgun/entities/container.py]: what is that one about?

Comment (abalakh) [reply airgun/entities/container.py]: [BZ1683348](https://bugzilla.redhat.com/show_bug.cgi?id=1683348) just some weird message appears, but container stops/starts successfully

Comment (oshtaier) [reply airgun/entities/container.py]: I think comment with BZ id will not harm much

File: airgun/entities/container.py [hunk 2]
```diff
@@ -37,4 +70,26 @@ class AddNewContainer(NavigateStep):
...
+    VIEW = ContainerDetailsView
+
+    def prerequisite(self, *args, **kwargs):
+        return self.navigate_to(self.obj, 'All')
+
+    def am_i_here(self, *args, **kwargs):
+        entity_name = kwargs.get('entity_name')
+        return (
+            self.view.is_displayed
+            and self.view.breadcrumb.read() == '{} - /{}'.format(
```
Comment (oshtaier) [inline airgun/entities/container.py]: strange construct =)

Comment (abalakh) [reply airgun/entities/container.py]: You don't say :)
That's how it looks on WebUI:
image

**Pull request merged**